src/sentry/integrations/client.py

```diff
@@ -135,7 +136,7 @@ def __init__(self, verify_ssl=True, logging_context=None):
     def get_cache_prefix(self):
         return "%s.client:" % six.text_type(self.integration_name)
 
-    def track_response_data(self, code, error=None):
+    def track_response_data(self, code, span, error=None):
```

Comment:
Is this function _only_ used on L241? I'm just thinking because there's no default for the `span` param being introduced it could throw some errors elsewhere

Reply:
@leeandher  This function is used in 5 places

Reply:
And there should always be the span because all the uses are wrapped within the 

```
with sentry_sdk.start_span(...) as span:
```

Reply:
@scefali I think the prev commit I was looking at only had one use, but cool beans if it's always provided a `span` then we're good